process/block/preprocess/smartContractResultPreProcessorFactory_test.go

```diff
@@ -1,10 +1,14 @@
 package preprocess_test
 
 import (
-	"github.com/multiversx/mx-chain-go/testscommon/common"
 	"testing"
 
+	"github.com/multiversx/mx-chain-go/testscommon/common"
```

Comment:
Bad imports formatting

Reply:
Fixed it in the next PR https://github.com/multiversx/mx-chain-go/pull/6676